responsive-desktop.css

```diff
@@ -0,0 +1,246 @@
+  }
+
+  .nav-items a {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+    padding: 0 12px;
+  }
+
+  .title,
+  .social,
+  .social-links {
+    padding-left: 260px;
+  }
+
+  .social-links {
+    mix-blend-mode: multiply;
+  }
+
+  .description {
+    padding: 0 260px;
+  }
+
+  .container {
+    display: flex;
+    column-gap: 0;
+    row-gap: 88px;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    width: 70%;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 0;
+    margin: 50px 100px;
+    height: fit-content;
+  }
+
+  .snapshot-details {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 544px;
+    height: fit-content;
+    align-items: flex-start;
+  }
+
+  .snapshot {
+    width: 554px;
+    border-radius: 1px;
+  }
+
+  #about-section {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    padding: 114px 96px;
+    width: 100%;
+    background-color: #fff;
+    border-top-right-radius: 150px;
+  }
+
+  .social-links2 {
+    padding-left: 0;
+  }
+
+  .social-links li {
+    display: inline;
+  }
+
+  .about-intro {
+    width: 40%;
+  }
+
+  .skilllist {
+    width: 60%;
+    padding: 24px;
+    margin: 0;
+    align-items: flex-start;
+    height: 409px;
+  }
+
+  .skilllist li {
+    padding: 0;
+    width: 100%;
+  }
+
+  .skilllist div {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .language-list {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .language-list li {
+    flex-direction: column;
+    height: fit-content;
+  }
+
+  .frameworks,
+  .skills {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding: 18px 0;
+    width: 100%;
+  }
+
+  .frameworks-title,
+  .skills-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+  }
+
+  #contact {
+    padding: 122px 200px;
+    align-items: center;
+  }
+
+  #contact h2 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    text-align: center;
+    color: #fff;
+    padding-top: 103px;
+    padding-bottom: 12px;
+    width: 447px;
+  }
+
+  .content {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    text-align: center;
+    padding: 0 24px 30px 24px;
+    width: 447px;
+    margin-left: 37px;
+  }
+
+  .contact-form {
+    padding-left: 12px;
+    width: 480px;
+  }
+
+  #username,
+  #user-email,
+  #text-area {
+    width: 100%;
+    margin: 12px 0;
+    border: 1px solid #cfd8dc;
+    padding: 1rem 12px;
+  }
```

Comment:
- [ ] Please notice in the figma template the border-raius for these inputs is 2px, currently you have it same as in the mobile version which is 10px, how about we fix this as well  :+1: